WordPress/src/main/java/org/wordpress/android/ui/posts/adapters/PostsListAdapter.java

```diff
@@ -368,6 +373,19 @@ public void onItemClick(AdapterView<?> parent, View view, int position, long id)
         listPopup.show();
     }
 
+    private void updatePostUploadProgressBar(ProgressBar view, PostModel post) {
+        if (UploadService.isPostUploadingOrQueued(post)) {
```

Comment:
Wow, I'm surprised to find there isn't a way to do it programmatically. I looked at the `ProgressBar` source code and there's no access to the `mBehavior` value :(